tasks/resolve_reference.rb

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require 'json'
+require 'aws-sdk-ec2'
+
+class AwsInventory < TaskHelper
+  attr_accessor :client
+
+  def config_client(opts)
+    return client if client
+
+    options = {}
+
+    if opts.key?(:region)
+      options[:region] = opts[:region]
+    end
+    if opts.key?(:profile)
+      options[:profile] = opts[:profile]
+    end
+    if opts[:credentials]
+      creds = File.expand_path(opts[:credentials])
+      if File.exist?(creds)
+        options[:credentials] = ::Aws::SharedCredentials.new(path: creds)
+      else
+        msg = "Cannot load credentials file #{opts[:credentials]}"
+        raise TaskHelper::Error.new(msg, 'bolt-plugin/validation-error')
+      end
+    end
+
+    ::Aws::EC2::Client.new(options)
```

Comment:
```suggestion
    Aws::EC2::Client.new(options)
```

I think we can probably get rid of the first scope since the SDK's class name no longer conflicts with the plugin's class name?

Reply:
oh, for sure. good catch I just copy pasted those.